docs/user_guide.md

```diff
@@ -1,19 +1,19 @@
 # Code issuer user guide
 
-This is a high level guide that can be used as a basic for creating a user guide
+This is a high-level guide that can be used as a basis for creating a user guide
 for an individual public health authority that is using the verification server.
 
 ## Account setup
 
 An administrator of your realm must invite you to join.
-Expect to receive an email for new account / password reset.
+Expect to receive an email for a new account / password reset.
 This will provide a link to set up your account password.
 
 ![new user](images/users/step02.png "Create a password")
 
 ### Second factor authentication
 
-On your next login, you will be given the option to enroll a second factor for authentication (SMS sent to your personal mobile phone). It is highly recommended to enroll the second factor.
+On your next login, you will be given the option to enroll a second factor for authentication (SMS sent to your mobile phone). It is highly recommended to enroll in a second factor. The phone number requires the international prefix (e.g. +1 for USA).
```

Comment:
I'm not sure if it's appropriate to put in USA specific advice. The overall idea was suggested by a few admins who noted they were confused by this input.

Reply:
On double-checking the form it looks like maybe the input widget was improved in a way that makes this change unneccessary. Happy to remove this if you agree.

Reply:
Let's remove. We added the nicer widget just this week https://github.com/google/exposure-notifications-verification-server/pull/702 because this is a known point of confusion

Reply:
We could also note that more-than-one second factor can be useful in case the user loses access to their primary device.